### PR TITLE
Filter to trigger builds on PRs that target a specific branch

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

File: vcs-utils/src/main/resources/buildServerResources/vcsSettings.jsp [hunk 1]
```diff
@@ -11,9 +11,9 @@
                     $j('#${keys.serverUrl}').show();
                     $j('#${keys.serverUrl} input').val('');
                     break;
-                case 'github':
+                case 'gilabelub':
```
Comment (jclariviere) [inline vcs-utils/src/main/resources/buildServerResources/vcsSettings.jsp]: ?

File: pullrequests/src/main/resources/buildServerResources/pullrequests.jsp [hunk 2]
```diff
@@ -29,6 +29,14 @@
 </script>
 
 <%@ include file="vcsSettings.jsp" %>
+<tr>
+    <th>Base branch:<l:star/></th>
```
Comment (jclariviere) [inline pullrequests/src/main/resources/buildServerResources/pullrequests.jsp]: Is there a way to have `master` by default?

**Comment by jclariviere** [comment]
LGTM

File: vcs-utils/src/main/resources/buildServerResources/vcsSettings.jsp (hunk 1 above)
Comment (olafleur) [inline vcs-utils/src/main/resources/buildServerResources/vcsSettings.jsp]: Seems like a renaming that went badly. ;-)

**Comment by olafleur** [comment]
LGTM

**Pull request merged**